src/mantarray_desktop_app/sub_processes/data_analyzer.py

```diff
@@ -660,9 +661,10 @@ def _start_recording_snapshot_analysis(self, recording_path: str) -> None:
             snapshot_dict = snapshot_dfs[0].to_dict()
             snapshot_list = [list(snapshot_dict[key].values()) for key in snapshot_dict.keys()]
             mag_analysis_msg = {"time": snapshot_list[0], "force": snapshot_list[1:]}
-
+        except UnableToConvergeError:
```

Comment:
We're already catching errors in `run_magnet_finding_alg`, I think we can just check the failed recordings instead of re-raising/catching this error

Reply:
Why is checking failed recordings better than just catching the exception itself?

Reply:
`try:
                ...
            except UnableToConvergeError:
                failed_recordings.append(
                    {
                        "name": recording_name,
                        "error": "Unable to process recording due to low quality calibration and/or noise",
                    }
                )
            except Exception:
                # Leaving in plain text because this error message is used directly in pop up modal to user when rec snapshot fails
                # it is never displayed to user in local analysis
                failed_recordings.append({"name": recording_name, "error": "Something went wrong"})`

Reply:
^^ @tannermpeterson